canonical-web-and-design/maas-ui
# fix(ui): display machine notifications on the summary only

## Done

- Move the machine notifications to only display on the summary page and update the component for use
with other tabs.

## QA

### MAAS deployment

To run this branch you will need access to one of the following MAAS deployments:

- [Karura](/HACKING.md#karura)
- [Bolla](/HACKING.md#bolla)
- [A development MAAS](/HACKING.md#development-deployment)
- [A local snap MAAS](/HACKING.md#snap-deployment) (this will not usually have machines)

### Running the branch

You can run this branch by:

- Serving with [dotrun](/HACKING.md#maas-ui-development-setup)
- [Building in a development MAAS](/HACKING.md#running-maas-ui-from-a-development-maas)

### QA steps

- View the summary tab for a machine, you should see notifications.
- Other tabs should show nothing.

## Fixes

Fixes: canonical-web-and-design/maas-ui#1851.
